redis-master/recipes/slave.rb

```diff
@@ -0,0 +1,26 @@
+passwordslave = Chef::EncryptedDataBagItem.load('Redis', 'redis', 'redis')['passwordslave']
```

Comment:
are we using this anywhere?

Reply:
Actually, while verification we need to use that.

Reply:
lets remove it from cookbook